app/models/rating.rb

```diff
@@ -0,0 +1,6 @@
+class Rating < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :resource
+  belongs_to :service
+  has_one :review
```

Comment:
this was incorrect, to be clear

belongs_to will place the foreign key in the declaring model whereas has_one will place it in the other model.

we definitely want rating to have a foreign key to user , not user having a foreign key to rating.